bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/io/ChannelPool.java

```diff
@@ -53,20 +54,22 @@
 
   private static final AtomicInteger ChannelIdGenerator = new AtomicInteger();
 
+  public static ImmutableList<InstrumentedChannel>
+      toInstrummentedChannels(List<ManagedChannel> channels) {
+    Preconditions.checkArgument(channels != null && !channels.isEmpty(),
+      "Channels have to be non-null and not empty.");
+    ImmutableList.Builder<InstrumentedChannel> response = ImmutableList.builder();
+    for (ManagedChannel channel : channels) {
+      response.add(new InstrumentedChannel(channel));
+    }
+    return response.build();
```

Comment:
nit: response is a bit misleading since it doesn't do any io, might be clearer to name it wrappedChannels or results